src/main/java/com/targomo/client/api/geo/AbstractGeometry.java

```diff
@@ -14,13 +14,15 @@
 
     private Integer crs;
     private String data;
+    private Boolean routeFromCentroid = true;
```

Comment:
I think we should use the primitive `boolean` here

Reply:
switched